api/server/runner_async_test.go

```diff
@@ -34,72 +29,3 @@ func testRouterAsync(ds models.Datastore, mq models.MessageQueue, rnr agent.Agen
 	s.bindHandlers(ctx)
 	return r
 }
-
-func TestRouteRunnerAsyncExecution(t *testing.T) {
```

Comment:
Is this functionality still available executing Fns? I don't think we test it elsewhere.